bundle/cmake-test/CMakeLists.txt

```diff
@@ -0,0 +1,41 @@
+project(serialbox2-cmake-test C CXX Fortran)
+enable_testing()
+cmake_minimum_required(VERSION 3.8.1)
+
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake")
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/../cmake")
+
+set(YODA_ROOT CACHE PATH "path to yoda package")
+
+if("${YODA_ROOT}" STREQUAL "")
+  message(FATAL_ERROR "YODA_ROOT not found! Try specifying it in the environment via -DYODA_ROOT=<>")
+endif()
+list(APPEND CMAKE_MODULE_PATH "${YODA_ROOT}/cmake")
+include(yodaInit)
+yoda_init()
+
+find_package(Serialbox CONFIG REQUIRED)

```

Comment:
This will load the `SerialboxConfig.cmake` file after installation. 
yoda has installed also a `SerialboxTargets.cmake` that contains all the exported targets, and is itself included from `SerialboxConfig.cmake`

Therefore after loading the config, can have in this CMake scope available all the serialbox targets. 
We print all the properties of these targets to verify that everything is well defined.